### ISSUE TITLE
Instruction pattern match constants

### ISSUE THREAD

**Issue by npmccallum** [enarx/enarx#328]
This is a debt bug after #326.

In `enarx-keep-sgx-shim`, we currently do something like this:

```rust
match core::slice::from_raw_parts(aex.gpr.rip as *const u8, 2) {
    // syscall
    [0x0f, 0x05] => { /* code here */ },

    // unknown invalid opcode
    _ => { /* code here */ }
}
```

The above code looks at the raw bytes for the instruction that produced the invalid opcode exception and then attempts to handle it. I don't know if this is possible, but it would be great to have constants for those match patterns.

**Comment by lkatalin** [comment]
Looks like this was addressed by #490 , should we close? @npmccallum

**Comment by npmccallum** [comment]
Yes.
